vdc/scripts/users/modify_users.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+op=$1
+
+while read user; do
+    stringarray=($user)
+    email=${stringarray[0]}
+    auth0_id=${stringarray[1]}
+    is_developer=${stringarray[2]}
+    is_sa=${stringarray[3]}
+    namespace=${stringarray[4]}
+    echo "email $email"
+    echo "auth0_id $auth0_id"
+    echo "is_developer $is_developer"
+    echo "is_sa $is_sa"
+    echo "namespace $namespace"
+    make user email="$email" user_id="$auth0_id" is_dev=$is_developer is_sa=$is_sa namespace="$namespace" op=$op
+done <users.txt
```

Comment:
where does this file come from?

Reply:
From the users.yml line. The list of user configurations is present in the secrets "users". This contains the a base64-encoded YAML string, containing the full resource definition for each user.